### ISSUE TITLE
[FP] CommonModuleInvalidType модуль ПолныеПрава всегда подсвечен как некорретный

### ISSUE THREAD

**Issue by sfaqer** [1c-syntax/bsl-language-server#2780]
## Диагностика
CommonModuleInvalidType 

## Версия
v0.20.0-rc.2

## Описание ложного срабатывания диагностики
Общий модуль с флагом "Привилегированный" не может быть с флагом "Клиент" как следствие не пролезает по таблице:
![image](https://user-images.githubusercontent.com/32082417/170190070-7c9c2289-a062-4b82-bf99-9d8ab6c24045.png)

**Comment by ivanmolodec** [comment]
Я бы уточнил что он не может быть с флагом "внешнее соединение", т.к. когда ставишь галку "привилегированный" то "внешнее соединение" автоматически отключается.

**Comment by sfaqer** [comment]
> Я бы уточнил что он не может быть с флагом "внешнее соединение", т.к. когда ставишь галку "привилегированный" то "внешнее соединение" автоматически отключается.

Ну это тогда не уточнение а дополнение, т.к с флагом Клиент обычное приложение он тоже быть не может

**Comment by ivanmolodec** [comment]
> > Я бы уточнил что он не может быть с флагом "внешнее соединение", т.к. когда ставишь галку "привилегированный" то "внешнее соединение" автоматически отключается.
> 
> Ну это тогда не уточнение а дополнение, т.к с флагом Клиент обычное приложение он тоже быть не может

Да, всё верно. Я только с УФ работаю и у меня на всех проектах стоит флаг "OrdinaryAppSupport": false, поэтому с таким поведением не встречался, сейчас открыл конфу в ОФ и там обе эти галки слетают при установке привилегированности.
